### ISSUE TITLE
[BUG]

### ISSUE THREAD

**Issue by JbvGaming** [robvdpol/RaceControl#271]
**Describe the bug**
Everytime I watch a stream (regular world feed), it just freezes the stream. No error, nothing. To fix it I have to close the stream window and reopen it. It doesn't happen in specific time frame and just happens randomly, at least it seems like it does. 

**To Reproduce**
1. Go to 'Live now'
2. Play stream 'World Feed'
3. Watch stream
4. Stream freezes

**Expected behavior**
Smooth stream with no freeze.

**Screenshots**

**Desktop (please complete the following information):**
 - Windows 11 Home
 - 64 bit
 - Version 21H2 / Build 22000.318

**F1TV account**
- F1TV Pro
- Germany
- Using VPN or proxy? No

**Additional context**

**Comment by SuRGeoNix** [comment]
Hi @JbvGaming, can you please check #270 and provide the requested information as it looks similar if not the same?

**Comment by JbvGaming** [comment]
@SuRGeoNix I actually looked at it before I submitted the bug, but for me it's not 10min in, it's random. Plus, for me it does not stutter at any time. It just freezes. Then I have just have to restart the stream not the whole app, since it doesn't crash.

**Comment by GHDutch** [comment]
To add, it does seem similar. Sometimes my whole app crashes, but in previous version I just could close the frozen stream and restart just that one. ;-) The crash only happened in latest version.

**Comment by SuRGeoNix** [comment]
@JbvGaming We will still need the requested information to check whether is flyleaf player's issue, hardware or RC's

**Comment by Crash1338** [comment]
Same here since the last 2 versions. Stream freezes (mostly after about 30-60 minutes) without any "warning" (no sound stutter or anything before). Difference to the original report is I'm on Windows 10 Pro 64bit (20H2)

**Comment by SuRGeoNix** [comment]
@Crash1338 v2.5.1 works fine with you and last 2 versions are 2.5.2 and 2.5.3 which freezes? Or you didn't try 2.5.3 yet?

**Comment by Crash1338** [comment]
Sorry, I mixed up my versions. I was using 2.5.3 just yesterday (which should be the latest one right now?). The version before had the same issue. The last version I know where I had no such problems was 2.4.0

**Comment by robvdpol** [comment]
@SuRGeoNix quite a lot of people reporting issues, is that because of switching to Vortice? Flyleaf 3.2.5 seemed to be much more stable.

**Comment by GHDutch** [comment]
> Sorry, I mixed up my versions. I was using 2.5.3 just yesterday (which should be the latest one right now?). The version before had the same issue. The last version I know where I had no such problems was 2.4.0

Same for me.

**Comment by SuRGeoNix** [comment]
@robvdpol Can't be sure, no much debugging / logs info, I'm just trying to guess again. We should consider adding more logging for the internal player. Can be RC integration with Flyleaf, can be Flyleaf itself, can be Vortice. FFmpeg's libraries didn't changed in 2.5.X right?

**Comment by jpd1968** [comment]
As I commented in #265 same issue here, World Feed live stream, internal player, RC version 2.5.3 screen freezes after a certain (random) period of time. Nothing in the log (as posted in #265 )

**Comment by JbvGaming** [comment]
> @JbvGaming We will still need the requested information to check whether is flyleaf player's issue, hardware or RC's

I will log everything if it happens again with the next Quali Session, but in FP3 today nothing happend and all went smoothly.

I looked into the Event Viewer in Windows but could not find anything new, just an error that happend on version 2.5.1 on the 7th of November. But since the player just freezes and not the whole program itself, I don't think that the windows error log should be of any help.

And I looked at the RC logs and couldn't find anything suspicious either from yesterday.
